bokeh/models/glyphs.py

```diff
@@ -393,15 +393,15 @@ class HBar(LineGlyph, FillGlyph, HatchGlyph):
     The y-coordinates of the centers of the horizontal bars.
     """)
 
-    height = NumberSpec(help="""
+    height = NumberSpec(default=value(1), help="""
```

Comment:
why the explicit `field` and `value` calls in this module? `NumberSpec` will interpret all these unambiguously

Reply:
I guess for consistency, but really I'm biased with writing bokehjs code where you have to use `value()` (well, `{value: }`).

Reply:
I would prefer to leave these out here, then